### PR TITLE
Post Demo Updates

### PR DESCRIPTION
**Pull request by simekanidvt**
In this PR i have fixed issues that i experienced during the demo and I implemented some recommendations that were given. 
 * Fixed Navigation from login to Landing page 
* Added an alert to login if the user enters incorrect details

### CODE REVIEW

File: WorkoutManager/WorkoutManager/Controllers/LoginViewController.swift [hunk 1]
```diff
@@ -22,12 +22,19 @@ class LoginViewController: UIViewController {
                 let username = userNameField.text else { return }
         
         if username == "Admin" && password == "TestPass123" {
+            print("herre")
```
Comment (bgouws) [inline WorkoutManager/WorkoutManager/Controllers/LoginViewController.swift:25]: This print statement can be removed 😄

**Review by bgouws — CHANGES_REQUESTED** [review]

**Review by kirsty14 — DISMISSED** [review]

**Review by smabhena — APPROVED** [review]

File: WorkoutManager/WorkoutManager/Controllers/LoginViewController.swift (hunk 1 above)
Comment (simekanidvt) [reply WorkoutManager/WorkoutManager/Controllers/LoginViewController.swift]: Missed that, done now 👍

**Review by kirsty14 — APPROVED** [review]
Looks good to me 👍🏻

**Review by bgouws — APPROVED** [review]
Looks good.

**Pull request merged**